cfme/cloud/provider/__init__.py

```diff
@@ -75,6 +75,23 @@ def is_displayed(self):
     including_entities = View.include(VMEntities, use_parent=True)
 
 
+class CloudProviderTenantsView(BaseLoggedInPage):
+    """
+    The collection page for cloud tenants
+    """
```

Comment:
Required:
Inline comment please. I think `docstring` not matching with intention means class name.

Reply:
I think you might have look over this view. I am suspect this view already available.
https://github.com/ManageIQ/integration_tests/blob/674b36716e2cd47632ff5ca0ac0b13d9467da2fe/cfme/cloud/tenant.py#L105

Reply:
@digitronik right...